metrix-mapping/src/test/java/com/powsybl/metrix/mapping/timeseries/FileSystemTimeseriesStoreTest.java

```diff
@@ -61,4 +68,142 @@ void testTsStore() throws IOException {
+        assertEquals(ts1, tsStore.getStringTimeSeries("ts1", 1).get());
+
+        // Case 2: some TimeSeries are asked
+        List<StringTimeSeries> stringTimeSeriesList = tsStore.getStringTimeSeries(Set.of("ts1", "ts2"), 1);
+        assertThat(stringTimeSeriesList).containsExactlyInAnyOrder(ts1, ts2);
+    }
+
+    @Test
+    void testDifferentKindOfTimeSeries() throws IOException {
+        // TimeSeries indexes
+        Instant now = Instant.ofEpochMilli(978303600000L);
+        RegularTimeSeriesIndex index = RegularTimeSeriesIndex.create(now, now.plus(2, ChronoUnit.HOURS), Duration.ofHours(1));
+
+        // TimeSeries
+        StoredDoubleTimeSeries ts1 = TimeSeries.createDouble("ts1", index, 1d, 2d, 3d);
+        StoredDoubleTimeSeries ts2 = TimeSeries.createDouble("ts2", index, 1d, 2d, 5d);
+        StringTimeSeries ts3 = TimeSeries.createString("ts3", index, "a", "b", "c");
+        StringTimeSeries ts4 = TimeSeries.createString("ts4", index, "a", "b", "c");
+
+        // TimeSeriesStore
+        FileSystemTimeseriesStore tsStore = new FileSystemTimeseriesStore(resDir);
+        tsStore.importTimeSeries(List.of(ts1, ts2, ts3, ts4), 1, false, true);
+        tsStore.importTimeSeries(List.of(ts1, ts2, ts3, ts4), 2, false, true);
+
+        // Case 1: a specific TimeSeries is asked
+        assertTrue(tsStore.getDoubleTimeSeries("ts1", 1).isPresent());
+        assertEquals(ts1, tsStore.getDoubleTimeSeries("ts1", 1).get());
+        assertTrue(tsStore.getStringTimeSeries("ts3", 1).isPresent());
+        assertEquals(ts3, tsStore.getStringTimeSeries("ts3", 1).get());
+
+        // Case 2: some TimeSeries are asked
+        List<DoubleTimeSeries> doubleTimeSeriesList = tsStore.getDoubleTimeSeries(Set.of("ts1", "ts2"), 1);
+        assertThat(doubleTimeSeriesList).containsExactlyInAnyOrder(ts1, ts2);
+        List<StringTimeSeries> stringTimeSeriesList = tsStore.getStringTimeSeries(Set.of("ts3", "ts4"), 1);
+        assertThat(stringTimeSeriesList).containsExactlyInAnyOrder(ts3, ts4);
+
+        // Case 3: all DoubleTimeSeries are asked
+        doubleTimeSeriesList = tsStore.getDoubleTimeSeries(1);
+        assertThat(doubleTimeSeriesList).containsExactlyInAnyOrder(ts1, ts2);
+
+        // Case 4: an absent TimeSeries is asked
+        assertTrue(tsStore.getDoubleTimeSeries("ts4", 1).isEmpty());
+    }
+
+    @Test
+    void testMultipleTimeSeries() throws IOException {
+        // TimeSeriesStore
+        FileSystemTimeseriesStore tsStore = new FileSystemTimeseriesStore(resDir);
+
+        // Add a file with multiple TimeSeries
+        Files.createDirectory(fileSystem.getPath("/tmp/ts1"));
+        Files.copy(Objects.requireNonNull(getClass().getResourceAsStream("/timeseries.json")),
+            fileSystem.getPath("/tmp/ts1/1"));
+
+        // An exception is thrown since there are mutiple TimeSeries in the file
+        PowsyblException exception = assertThrows(PowsyblException.class, () -> tsStore.getDoubleTimeSeries("ts1", 1));
```

Comment:
"ts1" here is the name of the directory. This could be misleading since there also exists a TimeSerie named "ts1" in the `timeseries.json` example file.

Could you rename either the directory or the timeserie in the file?